applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -836,7 +840,7 @@ void BaseSolidElement::CalculateOnIntegrationPoints(
                     CalculateKinematicVariables(this_kinematic_variables, point_number, this->GetIntegrationMethod());
 
                     // Compute material reponse
-                    CalculateConstitutiveVariables(this_kinematic_variables, this_constitutive_variables, Values, point_number, integration_points, GetStressMeasure());
+                    CalculateConstitutiveVariables(this_kinematic_variables, this_constitutive_variables, Values, point_number, integration_points, GetStressMeasure(), false);
```

Comment:
but then, don't you need to change all of the derived elements?

Reply:
I did it, i changed the UL, TL and small displ elements :)